src/cli_client/modes/outbound.js

```diff
@@ -33,7 +32,72 @@ const slackBroadcast = require('../extras/slack-broadcast')
 const TemplateGenerator = require('../utils/templateGenerator')
 const { TraceHeaderUtils } = require('ml-testing-toolkit-shared-lib')
 
-const bar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic)
+const totalProgress = {
+  totalTestCases: 0,
+  totalRequests: 0,
```

Comment:
Do you think the first two are adding much value here?.. can we have just the 3 - total, passed and failed assertions? what do you think

Reply:
Hi Sam, I am just maintaining those counts for future sake. I am not displaying them in the console for now.

Reply:
I am displaying only assertion counts in the console output like below.
```
[ 8 passed, 0 failed of 564 ]   P2P Transfer Happy Path -> Get party information
[ 19 passed, 0 failed of 564 ]  P2P Transfer Happy Path -> Send quote
[ 28 passed, 0 failed of 564 ]  P2P Transfer Happy Path -> Send transfer
[ 39 passed, 0 failed of 564 ]  Get Quotes -> Send quote
[ 39 passed, 10 failed of 564 ] Get Quotes -> Get Quotes
[ 50 passed, 10 failed of 564 ] Get Transfers -> Send quote
[ 59 passed, 10 failed of 564 ] Get Transfers -> Send transfer
[ 61 passed, 14 failed of 564 ] Get Transfers -> Get transfer
[ 66 passed, 15 failed of 564 ] Party info with missing header -> Party info with missing fspiop source header
[ 71 passed, 16 failed of 564 ] Party info with missing header -> Party info with missing date header
[ 76 passed, 17 failed of 564 ] Party info with missing header -> Party info with missing content header
[ 80 passed, 19 failed of 564 ] Party info with missing header -> Party info with missing accept header
[ 84 passed, 23 failed of 564 ] Party info of unprovisioned party -> Party info of unprovisioned party
[ 88 passed, 27 failed of 564 ] Party info of unprovisioned party -> Party info of unused type
[ 88 passed, 32 failed of 564 ] Party info with wrong header values -> Party info with wrong date header (BUG)
[ 88 passed, 37 failed of 564 ] Party info with wrong header values -> Party info with wrong content header(BUG)
```

Reply:
ok cool